ext4: Fix mount messages when quota disabled

From: Jan Kara <jack@suse.cz>

When quota is disabled, we should not print 'journaled quota not supported'
when user tried to mount non-journaled quota. Also fix typo in the message.

Signed-off-by: Jan Kara <jack@suse.cz>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
---
 fs/ext4/super.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: linux-2.6.25-git15/fs/ext4/super.c
===================================================================
--- linux-2.6.25-git15.orig/fs/ext4/super.c	2008-05-02 11:17:44.000000000 -0700
+++ linux-2.6.25-git15/fs/ext4/super.c	2008-05-02 11:17:52.000000000 -0700
@@ -1255,6 +1255,9 @@ set_qf_format:
 		case Opt_quota:
 		case Opt_usrquota:
 		case Opt_grpquota:
+			printk(KERN_ERR
+				"EXT4-fs: quota options not supported.\n");
+			break;
 		case Opt_usrjquota:
 		case Opt_grpjquota:
 		case Opt_offusrjquota:
@@ -1262,7 +1265,7 @@ set_qf_format:
 		case Opt_jqfmt_vfsold:
 		case Opt_jqfmt_vfsv0:
 			printk(KERN_ERR
-				"EXT4-fs: journalled quota options not "
+				"EXT4-fs: journaled quota options not "
 				"supported.\n");
 			break;
 		case Opt_noquota:
